### PR TITLE
[OSF-8011] Markdown image resize

### PR DESCRIPTION
**Pull request by adlius**
## Purpose

This PR allows users to specify image size using markdown syntax for wiki edits. See #7164 

## Changes

Changes are made to `markdown.js` to make markdown-it use the new plugin. Other related changes to add `makrdown-it-imsize` dependency to the project.

Note: Somehow, `{ node: 'empty' }` has to be added to the `module.export` for config files due to a webpack bug, otherwise wepack complains about not being able to resolve `fs`. (see: https://github.com/webpack-contrib/css-loader/issues/447). This could potentially be solved by upgrading to the latest version of webpoack.

## Side effects

None

## Ticket

https://openscience.atlassian.net/browse/OSF-8011

### CODE REVIEW

File: karma.common.conf.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ var webpackTestConfig = {
             // Assume test files are ES6
             {test: /\.test\.js$/, loader: 'babel-loader'},
         ])
+    },
+        node: {
+       fs: 'empty'
```
Comment (brianjgeiger) [inline karma.common.conf.js]: What does this do?

Comment (adlius) [reply karma.common.conf.js]: The added lines serves the same function for our JS tests as it does in `wepack.common.config.js` as a workaround to suppress an error due to a known webpack issue.

File: package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "lodash.includes": "^4.3.0",
     "lodash.set": "^4.3.2",
     "markdown-it": "~4.1.0",
+    "markdown-it-imsize": "^2.0.1",
```
Comment (binoculars) [inline package.json]: You must use `yarn add`/`yarn remove` to update package.json so that the yarn lockfile will be updated.

Comment (brianjgeiger) [reply package.json]: Yeah, this was a PR from September, so I think was from before yarn on the OSF. But we updated the yarn lock in another PR, so unless somethings changed there, this should be better.

**Pull request merged**